datasets/UKNI01_northern_ireland/UKNI01_northern_ireland.derived.ad

```diff
@@ -18,26 +18,25 @@
 - has_mountains = true
 - has_old_technologies = true
 - has_other = true
-- has_solar_csp = false
-- has_offshore_solar = false
```

Comment:
I don't think that the line `has_offshore_solar = false` should be deleted.

Reply:
Yes, true. But by default this attribute is set to false by @noracato, so luckily this is no issue